scripts/fonts.json

```diff
@@ -0,0 +1,450 @@
+{
+  "font-families": {
+    "Gothic A1": ["regular", "700"],
+    "M PLUS Rounded 1c": ["regular", "700"],
+    "Noto Sans": ["regular", "700", "italic", "700italic"],
+    "Noto Sans Arabic": ["regular", "700"],
```

Comment:
Consider [Noto Naskh Arabic](https://fonts.google.com/noto/specimen/Noto+Naskh+Arabic) as an analogue to Noto Sans Italic.

Reply:
I've added this font to the download, but whenever I try to use it, the script errors out with a duplicate code point error.  I've verified that I don't have any overlapping ranges so I'm not sure at this point how to move it forward.

Reply:
The code point range at issue is within the `[1536, 1791]` range.

Reply:
I can't even run the Naskh Arabic font through the pipeline by itself.  There seems to be something fundamentally wrong with this font that breaks this workflow.

Reply:
Hopefully we aren’t running into that classic Mapbox assumption that there’s a one-to-one correspondence between Unicode codepoints and glyph indices. We could compare this font against Noto Sans Arabic in a font editor and spot-check whether there are any discrepancies.

/ref https://github.com/maplibre/maplibre-gl-native/discussions/778#discussioncomment-5051041

Reply:
That does appear to be the case, though in this instance, the issue is manifesting in the `fonteditor-core` library.